/**:
  ros__parameters:
    DRIVE_LOOP_HZ: 50
    # if ACT_VALUE_TYPE: Throttle || Speed
    # elif ACT_VALUE_TYPE == Speed; network will predict speed
    # Throttle or Speed    
    ACT_VALUE_TYPE: "Throttle"
    TRANSMITTER_STICK_SPEED_MULTIPLIER: 2.5
    K_PID: {
      Kp: 0.2,
      Ki: 0.4,
      Kd: 0.001,
      I_max: 0.35
    }

    THROTTLE_MIN_PWM: 1350
    THROTTLE_MID_PWM: 1500
    THROTTLE_MAX_PWM: 1800
    # Full right
    STEERING_MIN_PWM: 1200
    STEERING_MID_PWM: 1540
    # Ful left
    STEERING_MAX_PWM: 1900
    ENCODER_UNITS: "m/s"
    ENCODER_TICKS_PER_UNIT: 1170